LondonBritishLibrary/orient/BLorient13228.xml

```diff
@@ -0,0 +1,142 @@
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 13228</idno>
+                        <altIdentifier><idno>Or. 13228</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 89</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT7287PrBaryaSusenyos"/>
+                            <note>Containing <ref type="work" corresp="LIT1763Legend"/>.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="omitted"/> ጸሎት፡ በእንተ፡ ሕማመ፡ ዓይነት፡ ባርያ፡ ወመጋኛ፡ ወሾቶላይ፡ ተያዥ፡ ወፃራሪ፡ ዛር፡ 
+                                ወተዋ<gap reason="illegible"/>ድድቅ፡ ወጋኔነ፡ ቀትር፡ በስመ፡ እግዚአብሔር፡ ሕያው፡ ነባቢ፡ ወተናጋሪ፡ ጸሎተ፡ ቅዱስ፡ 
+                                ሱ<choice><sic>ሱ</sic><corr>ስ</corr></choice>ንዮስ፡ በእንተ፡ አሰስሎ፡ ደዌ፡ እምሕፃናት፡ እለ፡ ደጠብው፡ ጥበ፡ እሞሙ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT7288PrBaryaDis"/>  
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="illegible"/> ጸሎት፡ በእንተ፡ ዓይነተ፡ ባርያ፡ መጋኛ፡ ወሾቶላይ፡ ውግዓት፡ ወቍርጥማት፡ ፍልፀት፡ 
+                                ወ<supplied reason="omitted" resp="PRS8999Strelcyn">ቍ</supplied>ርጸት፡ ቡዳ፡ ወጠቢብ። አአትብ፡ ገጽየ፡ ከመ፡ ኢይቅረብ፡ ደልፋኤል፡
+                                ሀልኤል፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT7290PrRheuma"/>  
+                            <incipit xml:lang="gez">ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርጥማት፡ ቀር<sic>ር</sic>ጥማትስ፡ ቀር<sic>ር</sic>ጥማትስ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <title ref="LIT7289PrChest"/>
+                            <incipit xml:lang="gez">ጸሎት፡ በእንተ፡ ሕማመ፡ ውግዓት፡ ሊቃኖስ፡ ብሂል፡ ዕፁብ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <title ref="LIT7291PrBaryaPhanuel"/>
+                            <note>Containing a salām to the archangel <persName ref="PRS7840Phanuel">Phanuel</persName>.</note>
```

Comment:
'Salām' can mean many things, so maybe it would be better to say:

```
Containing what 
<bibl><ptr target="bm:Strelcyn1978BritishLibrary"/>
<citedRange unit="page">136</citedRange></bibl>
defines as a "salām to the archangel <persName ref="PRS7840Phanuel">Phanuel</persName>.
```